geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/SimpleFeatureArrowFileWriter.scala

```diff
@@ -24,6 +24,7 @@ import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
   * For writing simple features to an arrow file.
   *
   * Uses arrow streaming format (no footer).
+  * Closing the SimpleFeatureArrowFileWriter closes the given os.
```

Comment:
@elahrvivaz do you believe this statement?  (I tried to trace through the indirection here, etc.)

Reply:
tracing through it looks like yes, although there appears to be a small possibility that an exception during the close would cause it to not be invoked: https://github.com/apache/arrow/blob/master/java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowWriter.java#L202

Reply:
Given that, would it be better to use a WithClose around the allocation of the outputstream as well?  Would closing the outstream multiple times cause trouble?

Reply:
hmm, generally i don't think closing multiple times would be an issue, but not sure if that's 100% true. probably any exceptions during the close would be infrequent and not result in massive memory leaks so maybe it's ok as is.